docs/js/RatingDocumentation.jsx

```diff
@@ -108,20 +108,264 @@ export default class RatingDocumentation extends Component {
+            <p style={ {marginTop: 0} }>
+              <i>Object</i>
+              <br />
+              optional
+            </p>
+            <p>
+              This style will be applied to the component if it is disabled.
+            </p>
+          </td>
+        </tr>
+        <tr>
+          <td style={ propertyNameStyle }>
+            disabledHoverStyle
+          </td>
+          <td style={ propertyDescriptionStyle }>
+            <p style={ {marginTop: 0} }>
+              <i>Object</i>
+              <br />
+              optional
+            </p>
+            <p>
+              This style will be applied to the component if it is disabled and it is hovered.
+            </p>
+          </td>
+        </tr>
+        <tr>
+          <td style={ propertyNameStyle }>
+            onMouseEnter
+          </td>
+          <td style={ propertyDescriptionStyle }>
+            <p style={ {marginTop: 0} }>
+              <i>Function</i>
+              <br />
+              optional
+            </p>
+            <p>
+              This callback will get executed as mouse enters into the component.
+            </p>
+          </td>
+        </tr>
+        <tr>
+          <td style={ propertyNameStyle }>
+            onMouseLeave
+          </td>
+          <td style={ propertyDescriptionStyle }>
+            <p style={ {marginTop: 0} }>
+              <i>Function</i>
+              <br />
+              optional
+            </p>
+            <p>
+              This callback will get executed as mouse leaves the component.
+            </p>
+          </td>
+        </tr>
+        <tr>
+          <td style={ propertyNameStyle }>
+            onMouseMove
+          </td>
+          <td style={ propertyDescriptionStyle }>
+            <p style={ {marginTop: 0} }>
+              <i>Function</i>
+              <br />
+              optional
+            </p>
+            <p>
+              This callback will get executed as mouse moves over the component.
+            </p>
+          </td>
+        </tr>
+        <tr>
+          <td style={ propertyNameStyle }>
+            onTouchStart
+          </td>
+          <td style={ propertyDescriptionStyle }>
+            <p style={ {marginTop: 0} }>
+              <i>Function</i>
+              <br />
+              optional
+            </p>
+            <p>
+              This callback will get executed as on touch start event.
+            </p>
+          </td>
+        </tr>
+        <tr>
+          <td style={ propertyNameStyle }>
+            onTouchMove
+          </td>
+          <td style={ propertyDescriptionStyle }>
+            <p style={ {marginTop: 0} }>
+              <i>Function</i>
+              <br />
+              optional
+            </p>
+            <p>
+              This callback will get executed as on touch move event.
+            </p>
+          </td>
+        </tr>
+        <tr>
+          <td style={ propertyNameStyle }>
+            onTouchEnd
```

Comment:
@jpuri In other components I only added the properties that are not common to the documentation & one sentence with the information that you as developer can add any other attribute that you normally use.

How do you feel about the documentation? Would you prefer to add all of the once we overwrite? What about we split between the custom ones & the common ones?